modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1579,44 +1579,6 @@ PERF_TEST_P_(CropPerfTest, TestPerformance)
     SANITY_CHECK_NOTHING();
 }
 
-//------------------------------------------------------------------------------
-
-PERF_TEST_P_(CopyPerfTest, TestPerformance)
-{
-    cv::Size sz_in = get<0>(GetParam());
-    MatType type = get<1>(GetParam());
-    cv::GCompileArgs compile_args = get<2>(GetParam());
-
-    initMatrixRandU(type, sz_in, type, false);
-    cv::Size sz_out = sz_in;
-
-    // OpenCV code ///////////////////////////////////////////////////////////
-    cv::Mat(in_mat1).copyTo(out_mat_ocv);
-
-    // G-API code ////////////////////////////////////////////////////////////
-    cv::GMat in;
-    auto out = cv::gapi::copy(in);
-    cv::GComputation c(in, out);
-
-    // Warm-up graph engine:
-    auto cc = c.compile(descr_of(gin(in_mat1)),
-                        std::move(compile_args));
-    cc(gin(in_mat1), gout(out_mat_gapi));
-
-    TEST_CYCLE()
-    {
-        cc(gin(in_mat1), gout(out_mat_gapi));
-    }
-
-    // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
-    EXPECT_EQ(out_mat_gapi.size(), sz_out);
-
-    SANITY_CHECK_NOTHING();
-}
-
-//------------------------------------------------------------------------------
```

Comment:
I'd keep it, at least for the CPU backend instantiation. Because, hm, why not.

Reply:
Applied